docs/CHANGELOG.asciidoc

```diff
@@ -195,6 +195,8 @@ To display the cluster data in *Discover*, load documents directly from `_source
 [[known-issues-v7.12.0]]
 === Known issues
 
+*Saved object migrations*
```

Comment:
Formatting probably needs help. I tried to make these subheadings `====` but build didn't allow